### ISSUE TITLE
Crash STPAddCardViewController with scan.io

### ISSUE THREAD

**Issue by mwoollard** [stripe/stripe-ios#1356]
## Summary
Scan Card button remains active whilst card being added, tapping can cause crash

## Steps to reproduce
App using scan.io integration w/ STPAddCardViewController
From STPAddCardViewController tap Scan button
Scan a card
Whilst card is being added (network indicator top right) tap Scan button again

## Expected behaviour
Button is disabled

## Actual behaviour
Button is active and tapping either re-opens scan.io UI or causes app crash

## iOS version
12.4.1

## Installation method
Cocoapods

## SDK version
16.0.7

## Other information
Not a great deal of info but crash stack, the crash occurs if the card add network call completes whilst the scan.io controller is being presented. If it has completed presentation it gets dismissed along with the STPAddCardViewController when network call completes.
```
Crashed: com.apple.main-thread
EXC_BAD_ACCESS KERN_INVALID_ADDRESS 0x0000000000000010

UIKit
__56-[UIPresentationController runTransitionForCurrentState]_block_invoke + 460
1
UIKit
_runAfterCACommitDeferredBlocks + 292
9 arrow_drop_down
UIKit
UIApplicationMain + 236
10
Pilkington-Stage
AppDelegate.swift - Line 20
main + 20
11
libdyld.dylib
start + 4
```

**Comment by yuki-stripe** [comment]
The fix to hide the scan button is in master.  Thanks for the report @mwoollard!

**Comment by davidme-stripe** [comment]
This has been fixed in Stripe SDK 17.0.1.
